main/src/main/java/de/blinkt/openvpn/core/OpenVpnManagementThread.java

```diff
@@ -134,8 +140,12 @@ public void run() {
 
 
             // Close the management socket after client connected
+            try {
+                mServerSocket.close();
+            } catch (IOException e){
+                VpnStatus.logException(e);
+            }
 

```

Comment:
Did you test that on older Android versions? (E.g. 4.0, 4.1). I am not really sure but I remember something about closing that socket too early brought problems.